Volume/Views/MainView/MagazinesList.swift

```diff
@@ -2,105 +2,32 @@
 //  MagazinesList.swift
 //  Volume
 //
-//  Created by Hanzheng Li on 3/4/22.
-//  Copyright © 2022 Cornell AppDev. All rights reserved.
+//  Created by Vin Bui on 3/18/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
```

Comment:
the disrespect 😂

Reply:
i rewrote a lot of stuff so whoever works on this next can blame me if it doens't work